src/components/RecordingDialog.tsx

```diff
@@ -122,7 +120,6 @@ const RecordingDialog = ({
       onClose={handleOnClose}
       size="lg"
       isCentered
-      finalFocusRef={finalFocusRef}
```

Comment:
@microbit-grace I don't think this is required as focus is naturally returned to the record button in the case of clicking or keyboard action. Correct me if I'm wrong.

Reply:
Yes, it seems to naturally return to the record button. It didn't seem to be the case before, not sure what has changed, but good to remove it.

I have noticed that it doesn't return for the very first recording, but I think the issue exists in the add-data branch as well. I will note it in the add-data PR description.